tfhe-zk-pok/src/proofs/pke_v2.rs

```diff
@@ -513,6 +518,13 @@ pub fn compute_crs_params(
     // Since we use B_bound_square, the log is divided by 2
     let m_bound = 1 + ceil_ilog2(B_bound_squared).div_ceil(2) as usize;
 
+    // m_bound is used to do the bit decomposition of a u64 integer, so we check that it can be
+    // safely used for this
+    assert!(
+        m_bound <= 64,
+        "Invalid parameters for zk_pok, B_squared: {B_squared}, d: {d}, k: {k}, m_bound: {m_bound} > 64"
```

Comment:
same here for the m_bound could be "we only support 64 bits integer, the computed m parameter is {m_bound} bits > 64 bits"